### ISSUE TITLE
[FR] Keep ansi code with '--with-nth' 

### ISSUE THREAD

**Issue by rockandska** [junegunn/fzf#1748]
- [X] I have read through the manual page (`man fzf`)
- [X] I have the latest version of fzf
- [X] I have searched through the existing issues

## Info

- OS
    - [X] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Shell
    - [X] bash
    - [ ] zsh
    - [ ] fish
    
## Problem / Steps to reproduce

Will be really useful to keep the ansi codes when using '--with-nth' option

At that time, using '--with-nth' remove every characters from the left of the separator letting previously colors applied removed from the output

For example:
```
echo -ne '\e[31m/a\e[33m/file\e[0m\n/a/test' | fzf --ansi --with-nth=3.. -d '/'
```
Here, `file` lost its yellow color

With the preservation of ansi colors with `--with-nth=` , the string will preserve its colors and will be print as :
```
echo -ne '\e[31m\e[33mfile\e[0m\ntest'
```
So the last color applied will be preserved as in the original input

Regards,

**Comment by junegunn** [comment]
Are you sure that you have the latest version of fzf? It's fixed several months ago (#1500).

**Comment by rockandska** [comment]
Sorry, I indeed missed a release and didn't seen the previous issue about this.
Seems to works like a charm and [fzf-obc](https://github.com/rockandska/fzf-obc) will be happy to benefit of this :)
